python/cuml/dask/DGX.md

```diff
@@ -0,0 +1,112 @@
+# Using IB for Dask + cuML MG on a DGX
```

Comment:
Probably would rename this file to `BUILD-DGX.md` or something along those lines for clarity. Also it would be good to have a link to this (buried) doc from the base `BUILD.md`, perhaps a section on Dask/Networking there?

Reply:
I think that's a good idea. The description in this PR actually contains the most recent version of this markdown file. I just found it easier to modify it there and instantly see the changes, then copy it back into the branch when it's done. 

I'm thinking we might want to build a directory structure, perhaps in the repository root or in the docs dir, to put many of our informational markdown files together so they don't get scattered around the codebase.

One of the reasons I have not created the PR for the algorithm guidelines / checklists is because I'm not completely sure where it belongs and I'm concerned about continuing the add markdown files directly to the repository root. 

I'd say we should use Github's wiki option but I'd prefer if the documentation could be released & shipped with the codebase so that it always stays relevant to particular versions, rather than being updated and getting out of date as new versions are released. 

What do you think about a simple directory structure like this?

`wiki/python/`
`wiki/cpp/`
`wiki/multigpu/`
...

I can file a separate issue for this, but wanted to gather thoughts first. cc @JohnZed @teju85 @Salonijain27 @oyilmaz-nvidia ?

Reply:
For reference, [here](https://github.com/facebookresearch/faiss/wiki) is an example of the Github Wiki used by the FAISS project. Notice they are not able to be tied to versions.

Reply:
Another option could be to start incorporating these types of things into our docs, similar to [Dask](https://docs.dask.org/en/latest/). As @divyegala pointed out yesterday in #854, it is always nice to provide detailed copy/paste examples in some form. I think this DGX setup would be better geared towards that.

Reply:
@cjnolet I think your proposed directory structure sounds good! 

I think the ideal path would be to have it in the wiki and in the docs. The wiki not being tied to a version is not a big deal, since what we want there is the latest version for developers to use, and since we're gonna be using it ourselves it should be straightforward to keep it up to date. I think its worth having the instructions in the repo/wiki and not only in the docs site, what do you think?